radicle-node/src/worker.rs

```diff
@@ -327,7 +327,9 @@ impl<G: Signer + Ecdh + 'static> Worker<G> {
         if result.success() {
             log::debug!(target: "worker", "Fetch for {} exited successfully", rid);
             let head = repo.set_head()?;
-            log::debug!(target: "worker", "Head for {} set to {head}", rid);
+            log::debug!(target: "worker", "HEAD for {} set to {head}", rid);
```

Comment:
It's not just `HEAD`, it's also the default branch ref :)

Reply:
Ah fair enough! I'll revert :)